prow/plugins/override/override.go

```diff
@@ -334,15 +335,31 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent,
 		return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, resp))
 	}
 
+	baseSHAGetter := shaGetterFactory(oc, org, repo, pr.Base.Ref)
+	presubmits, err := oc.presubmits(org, repo, baseSHAGetter, sha)
+	if err != nil {
+		msg := "Failed to get presubmits"
+		log.WithError(err).Error(msg)
+		return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, msg))
+	}
+
 	contexts := sets.NewString()
 	for _, status := range statuses {
 		if status.State == github.StatusSuccess {
 			continue
 		}
+
 		contexts.Insert(status.Context)
+
+		if result := funk.Find(presubmits, func(job config.Presubmit) bool {
+			return job.Context == status.Context
+		}); result != nil {
+			contexts.Insert(result.(config.Presubmit).Name)
+		}
 	}
+
 	if unknown := overrides.Difference(contexts); unknown.Len() > 0 {
-		resp := fmt.Sprintf(`/override requires a failed status context to operate on.
+		resp := fmt.Sprintf(`/override requires a failed status context or a Prow Job name to operate on.
```

Comment:
```suggestion
		resp := fmt.Sprintf(`/override requires a failed status context or a job name to operate on.
```

Prowjobs are the custom resource instances and they have a randomized name so this is slightly confusing right now